db/organizations/zeotap.eno

```diff
@@ -0,0 +1,6 @@
+name: zeotap
```

Comment:
```suggestion
name: Zeotap
```

Reply:
Same here in their own Kununu presence: https://www.kununu.com/de/zeotap

Their homepage is inconsistent, but looks like you are right that it is official lowercase.